java/src/jmri/jmrix/swing/ConnectionsPreferencesPanel.java

```diff
@@ -40,7 +40,7 @@
  */
 public class ConnectionsPreferencesPanel extends JTabbedPane implements ManagingPreferencesPanel {
 
-    private static final ResourceBundle rb = ResourceBundle.getBundle("apps.AppsConfigBundle"); // NOI18N
+    private static final ResourceBundle rb = ResourceBundle.getBundle("apps.AppsConfigBundle"); // NOI18N for some items
```

Comment:
The `//NOI18N` comment should remain as-is this is used by several IDE's to flag a specific text string as not being for internationalisation.

Changing this comment, _might_ break that functionality.